src/ddmd/statement.d

```diff
@@ -296,10 +332,13 @@ extern (C++) abstract class Statement : RootObject
      * If this statement has code that needs to run in a finally clause
      * at the end of the current scope, return that code in the form of
      * a Statement.
-     * Output:
-     *      *sentry         code executed upon entry to the scope
-     *      *sexception     code executed upon exit from the scope via exception
-     *      *sfinally       code executed in finally block
+     * Params:
+     *    sc = context
+     *    sentry     = set to code executed upon entry to the scope
+     *    sexception = set to code executed upon exit from the scope via exception
+     *    sfinally   = set to code executed in finally block
```

Comment:
Not for the functionality. But it matters for consistency and that's important for readability.